tests/test_type_inference/test_function_def_inference.py

```diff
@@ -13,6 +13,14 @@ def _parse_to_function(function_name, args_list, return_statement):
            f'    return {return_statement}'
 
 
+def _parse_to_annotated_function(function_name, args_list, return_statement, annotations):
```

Comment:
It is a poor choice to create more of these methods. Use the new approach of creating an actual astroid node instead.